packages/common/src/Adaptor.js

```diff
@@ -932,3 +932,53 @@ export function assert(expression, errorMessage) {
     return state;
   };
 }
+
+/**
+ * Outputs a message to the console while maintaining the state.
+ * @public
+ * @function
+ * @example <caption>Log specific values</caption>
+ * log('Hello', 'World')
+ * // Logs: Hello World
+ * log($.user.name)
+ * // Logs value of state.user.name
+ * @example <caption>Log a specific object from state</caption>
```

Comment:
This is a strange example. If you want to show how to log state without including credentials (and I don't think you need to do that?), then you should name the example that.

Because if I wanted to "log a specific value", I'd probably do `log($.data)`. I wouldn't use all that convoluted spread stuff.

Reply:
🤔 , Right right
Okay i have removed the example for logging state without credential, We still have an example for log message from state, That should be enough